assets/openshift-controller-manager/config.yaml

```diff
@@ -25,3 +25,4 @@ servingInfo:
   - TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
   - TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305
   - TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
+minTLSVersion: VersionTLS12
```

Comment:
The spacing here doesn't look correct and I suspected would cause the OpenShift controller manager to crash.  However, in my testing it didn't.  So I started hacking up this file and to my surprise, I could add bogus options and it didn't care.  This makes me wonder if we have bigger problems here.  Please investigate this further to determine if this change is being honored.

Reply:
Evan and I investigated further and believe that unknown keys are ignore but unknown values for known keys are properly verified.  Therefore, all this PR should need to do is fix the spacing on this option.